charts/caraml-authz/templates/bootstrap-policy-job.yaml

```diff
@@ -4,10 +4,6 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: {{ template "caraml-authz.resource-prefix" . }}-bootstrap-policy
-  annotations:
```

Comment:
will this have the job restart a few times possibly? Because it needs the authz pod to be up and running to execute it?

Reply:
Yes the pod will keep restarting until it succeeds. Have tested in caraml umbrella chart